src/main/java/org/jabref/gui/desktop/os/Linux.java

```diff
@@ -30,7 +30,11 @@ public void openFile(String filePath, String fileType) throws IOException {
             viewer = "xdg-open";
         }
         String[] cmdArray = { viewer, filePath };
-        Runtime.getRuntime().exec(cmdArray);
+        Process p;
```

Comment:
In java you don't need to declare and assign variables separately. So the following works (and is slightly preferred): 
```suggestion
        Process p = Runtime.getRuntime().exec(cmdArray);
```